### ISSUE TITLE
Data isn't sent to ElasticSearch when using Assertion Response

### ISSUE THREAD

**Issue by rondevops34** [delirius325/jmeter-elasticsearch-backend-listener#44]
Hi,
When using assertion response the data isn't sent to Elastic Search.
Please advise,
Thanks in advance,
Ron

**Comment by delirius325** [comment]
@rondevops34 Could you please elaborate on your issue. Add screenshots, logs, etc. 

Thanks,

**Comment by rondevops34** [comment]
Hi,
After investigating the issue again I found out that the problem was that the mapping of the Assertion Results field in my index in ES was incorrect. I defined the field correctly and it looks fine.
However,in Kibana  in the discover screen when viewing the Assertion Results field record in table format I see yellow triangle. When I click on it I see the message "Objects in arrays are now well supported"
Despite the above mark and message the data regarding Assertion Results is accurate in case of success or failure.
